config/v1.5/calico.yaml

```diff
@@ -0,0 +1,632 @@
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: calico-node
+  namespace: kube-system
+  labels:
+    k8s-app: calico-node
+spec:
+  selector:
+    matchLabels:
+      k8s-app: calico-node
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate:
+      maxUnavailable: 1
+  template:
+    metadata:
+      labels:
+        k8s-app: calico-node
+      annotations:
+        # This, along with the CriticalAddonsOnly toleration below,
+        # marks the pod as a critical add-on, ensuring it gets
+        # priority scheduling and that its resources are reserved
+        # if it ever gets evicted.
+        scheduler.alpha.kubernetes.io/critical-pod: ''
+    spec:
```

Comment:
I think you should add `priorityClassName: system-node-critical` to the `calico-node` daemonset spec with their permission.

Reply:
@2ffs2nns @caseydavenport `scheduler.alpha.kubernetes.io/critical-pod: ''` is being [deprecated in 1.13](https://kubernetes.io/docs/tasks/administer-cluster/guaranteed-scheduling-critical-addon-pods/#marking-pod-as-critical), are you guys fine with me adding `priorityClassName: system-node-critical` to the spec instead?

Reply:
Yep - that SGTM